Library/Models/SurveyState.cs

```diff
@@ -20,7 +20,7 @@ namespace Nfield.Models
     /// <summary>
     /// The fieldwork status of the survey
     /// </summary>
-    public enum SurveyStatus
+    public enum SurveyState
```

Comment:
You can't do this, this is a breaking change.

Reply:
What is it breaking? Its an internal model and all it's usages have been updated. At the end it's parsed into the same that SurveyStatus

Reply:
You mean someone which was expecting a Survey.Status in the response and now that code it broken?

Reply:
it's public, not internal. so someone could have used it